src/pages/workspaces/workspace/Analyses.js

```diff
@@ -145,70 +138,49 @@ const AnalysisCard = ({ namespace, name, updated, metadata, toolLabel, listView,
   }, [
     h(Link, { 'aria-label': 'Analyses menu', onClick: e => e.preventDefault() }, [
       icon('ellipsis-v', {
-        size: listView ? 18 : 16
+        size: analysisContextMenuSize
       })
     ])
   ])
 
-  const title = div({
+  //the flex values for columns here correspond to the flex values in the header
+  const artefactName = div({
     title: getDisplayName(name),
-    style: _.merge({
-      ...Style.elements.card.title, whiteSpace: 'normal', overflowY: 'auto'
-    }, listView ? {
-      marginLeft: '1rem', flexGrow: 1
-    } : { height: 60, padding: '1rem' })
-  }, getDisplayName(name))
+    style: {
+      ...Style.elements.card.title, whiteSpace: 'normal', overflowY: 'auto', flex: 5, textAlign: 'left'
+    }
+  }, [getDisplayName(name)])
+
+  const toolIconSrc = Utils.switchCase(application,
+    [tools.Jupyter.label, () => jupyterLogo],
+    [tools.RStudio.label, () => rLogo])
+  const toolIcon = div({ style: { marginRight: '1rem' } }, [
+    img({ src: toolIconSrc, style: { height: 40, width: 50 } })
+  ])
 
-  const appIconSrc = Utils.switchCase(toolLabel, [tools.Jupyter.label, () => jupyterLogo], [tools.RStudio.label, () => rLogo])
-  const appIcon = div({ style: { marginRight: '1rem' } }, [
-    img({ src: appIconSrc, style: { height: 40, width: 40 } })
+  const toolContainer = div({ style: { display: 'flex', flex: 1, flexDirection: 'row', alignItems: 'center' } }, [
+    toolIcon,
+    // this is the tool name, i.e. 'Jupyter'. It is named identical to the header row to simplify the sorting code at the cost of naming consistency.
+    application
```

Comment:
```suggestion
  const toolContainer = div({ style: { display: 'flex', flex: 1, flexDirection: 'row', alignItems: 'center' } }, [
    img({ src: logo, style: { marginRight: '1rem', height: 40, width: 50 } }),
    // this is the tool name, i.e. 'Jupyter'. It is named identical to the header row to simplify the sorting code at the cost of naming consistency.
    application
  ])
```

Where `logo` is derived from the application. Perhaps something like `tools[application]?.icon`

Reply:
I wanted to make it explicit everywhere since there are multiple versions of each. There's an RLogo, an RStudio logo, and an R bioconductor logo which will be used in different places. Theres also a square jupyter logo with no text, and a longer rectangular one, both with different use cases